src/main/java/com/sprints/services/SprintsServiceImpl.java

```diff
@@ -88,6 +88,11 @@ public String createSprint(SprintDomain sprintDomain) {
 			Sprint sprint = sprintsValidationsRepository.oneSprintBacklogValidation();
 			sprintsValidations.sprintValidateInBacklog(sprint);
 		}
+		
+		if(sprintFinal.getActive()==true && sprintFinal.getIs_backlog() == true) {

```

Comment:
Please check the sprint that you are validating, sprintFinal is the sprint that is coming from DB, and you need validate the sprint that is coming from the controller